python/its-iqm/its_iqm/central/file.py

```diff
@@ -0,0 +1,65 @@
+# Software Name: its-iqm
+# SPDX-FileCopyrightText: Copyright (c) 2023 Orange
+# SPDX-License-Identifier: MIT
+# Author: Yann E. MORIN <yann.morin@orange.com>
+
+from __future__ import annotations
+import configparser
+import logging
+import threading
+import time
+
+
+class Central:
```

Comment:
Be careful to not confuse the future Centrel Authority with the central geoservervice (in opposite of a potential edge geoservice).

Reply:
I renamed _central_ to _authority_, both in filenames and code.